src/store/student-store.js

```diff
@@ -42,4 +42,10 @@ store.registerHandler('CHECKIN_SUCCESS', data => {
     store.emitChange();
 });
 
+
+store.registerHandler('ITEM_DUEDATE_UPDATED', data => {
+    student = data;
```

Comment:
Dude, really? It says(in the docs/Actions.md) it will return a student object.
https://github.com/TheFourFifths/consus-client/pull/91/files/e8d16255bdf90848555d5184af2b316b9a678eca#diff-70fa60644f355590424cba92a4f362f4R235 in case you can't find it :)